src/ensembl/src/content/app/custom-download/containers/header/customDownloadHeaderHelper.ts

```diff
@@ -75,16 +75,26 @@ export const getEndpointUrl = (
   };
 
   // FIXME: Temporarily apply the filters locally
-  const gene_ids = get(processedFilters, 'genes.limit_to_genes', [])
-    .join(',')
-    .split(',')
-    .map(trim)
-    .filter(Boolean);
+  const gene_ids: string[] = [];
+
+  const limitToGenes = get(processedFilters, 'genes.limit_to_genes', []);
```

Comment:
What does `limitToGenes` mean here? Is it a limit to the number of genes?

Reply:
This is just a temporary solution so I have just used the field name displayed in the accordion.